includes/template-library/sources/local.php

```diff
@@ -1293,11 +1293,31 @@ public function maybe_render_blank_state( $which ) {
 			}
 
 			$current_type = 'template';
-
-			$inline_style .= '#elementor-template-library-tabs-wrapper {display: none;}';
+			$additional_inline_style = '#elementor-template-library-tabs-wrapper {display: none;}';
 		}
 
+		$this->render_blank_state( $current_type, $additional_inline_style );
+	}
+
+	public function render_blank_state( $current_type, $additional_inline_style = '', $custom_description = '', $custom_href = '' ) {
```

Comment:
Use one parameter `$args` and parse default args.

Reply:
Handled: https://github.com/elementor/elementor/pull/13037